typescript/src/schema/api.ts

```diff
@@ -52,4 +52,11 @@ export class API {
       proto => this.protos[proto].fileToGenerate
     );
   }
+
+  get protoFilesToGenerate(){
+    return this.filesToGenerate.map(function(file){
+      return '\"../../protos/' + file + '\"' + ',\n';
```

Comment:
Please use `\`"../../protos/${file}"` instead of concatenation. Also, it makes sense to use `.join` for `',\n'` in the template (and not here), because commas and newlines are presentation and belong to a template.